build/dev/probe_local_instance.sh

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env bash
+
+set -eo pipefail
+
+echo "Re/Create probe_local_instance_test_result file"
+RESULTFILE="$(pwd)/probe_local_instance_test_result"
+touch "${RESULTFILE}"
+cat /dev/null > "${RESULTFILE}"
+declare -a localhost_containers=("dss_sandbox_local-dss-http-gateway_1" 
```

Comment:
I think we may set ourselves up for easier extension of this script in the future if we set variables for all the container names and then just use those variables.  So, something like:

```bash
GATEWAY_CONTAINER="dss_sandbox_local-dss-http-gateway_1"
OAUTH_CONTAINER="dss_sandbox_local-dss-dummy-oauth_1"
localhost_containers="$GATEWAY_CONTAINER $OAUTH_CONTAINER"
```

...and then we can reuse those variables again later:

```bash
docker run --network dss_sandbox_default \
  --link $OAUTH_CONTAINER:oauth \
	--link $GATEWAY_CONTAINER:local-gateway \
```

Note that I think we only have to check for these two containers because they're the only ones we talk to.  If the CRDB or gRPC containers are missing, that still might possibly be ok since we don't talk to them directly.